### ISSUE TITLE
Framework: Calypso doesn't fully work when running docker locally

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-calypso#11523]
When running Calypso in docker locally there are some things that don't work. This makes it hard to run the e2e automated tests against a local docker environment as they fail.

1. The accept-invite screen was blank (no console errors) 
<img width="1003" alt="accept invite" src="https://cloud.githubusercontent.com/assets/128826/22363856/06bc01ba-e4b9-11e6-8131-5f9f0bceb307.png">
2. Domains page search results during signup (/start) was blank (no console errors) 
<img width="938" alt="domains" src="https://cloud.githubusercontent.com/assets/128826/22363864/0e190994-e4b9-11e6-9caf-de82c0b0d90a.png">
3. Customizer was really slow, slow enough to not be able to run the e2e tests - I'm not sure whether this is something to do with iFraming the hosted site

I am not sure if these are issues running with host files etc

**Comment by gwwar** [comment]
I'll see if I can find any more info here

**Comment by gwwar** [comment]
ah @ockham do you recall if there are any limitations for testing SSR pages in a docker container?

**Comment by gwwar** [comment]
So I verified the issue above. We don't see any suggestions in the local docker image. Why this works in production but not here is a mystery. @umurkontaci any ideas on why this is the case? I'd probably need to take another day to recall how the flow actually works in the component.

Domains suggestions looks half-ported to redux, but I never saw a related redux event fire locally or in the docker image and related state remained empty. (Maybe I only implemented it for the suggestions nudge?). The logic in the domain step is overall very complex. I'd recommend adding suggestions to the data layer and simplifying the current redux state, so we can have a better sense of what's happening. (Versus a bare wpcom lib call in the component.)

![screen shot 2018-01-11 at 12 39 55 pm](https://user-images.githubusercontent.com/1270189/34847265-941585bc-f6cf-11e7-8993-43911a5fcc3f.png)
